vcd/datasource_vcd_org_vdc_test.go

```diff
@@ -67,8 +67,8 @@ func TestAccVcdVdcDatasource(t *testing.T) {
 						"data."+datasourceVdc, "compute_capacity.0.cpu.0.allocated", "vcd_org_vdc."+vdcName, "compute_capacity.0.cpu.0.allocated"),
 					resource.TestCheckResourceAttrPair(
 						"data."+datasourceVdc, "compute_capacity.0.cpu.0.limit", "vcd_org_vdc."+vdcName, "compute_capacity.0.cpu.0.limit"),
-					resource.TestCheckResourceAttrPair(
-						"data."+datasourceVdc, "compute_capacity.0.cpu.0.overhead", "vcd_org_vdc."+vdcName, "compute_capacity.0.cpu.0.overhead"),
+					resource.TestMatchResourceAttr(
+						"data."+datasourceVdc, "compute_capacity.0.cpu.0.overhead", regexp.MustCompile(`^\d*$`)),
```

Comment:
```suggestion
						"data."+datasourceVdc, "compute_capacity.0.cpu.0.overhead", regexp.MustCompile(`^\d+$`)),
```
Your regexp can also match the empty string.

Reply:
updated all occurences